.gitignore

```diff
@@ -25,3 +25,4 @@ kube/services/fluentd/dockerlogs/
 *.cache
 *pyc
 !doc/terraform
+*.json
```

Comment:
We might not want to add this to gitignore, this repo has a few .json files and we might add/remove/edit  anytime.